src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/LeggTilBrevmottakerModalFagsak.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { LeggTilBrevmottakerModal } from './LeggTilBrevmottakerModal';
+import type { BrevmottakerUseSkjema, SkjemaBrevmottaker } from './useBrevmottakerSkjema';
+import { felterTilSkjematBrevmottaker } from './useBrevmottakerSkjema';
+import { useFagsakContext } from '../../../../../context/fagsak/FagsakContext';
+
+interface IFagsakModalProps {
+    lukkModal: () => void;
+}
+export const LeggTilBrevmottakerModalFagsak: React.FC<IFagsakModalProps> = ({ lukkModal }) => {
+    const { manuelleBrevmottakerePåFagsak, settManuelleBrevmottakerePåFagsak } = useFagsakContext();
+
+    const lagreMottaker = (useSkjema: BrevmottakerUseSkjema) => {
+        if (useSkjema.kanSendeSkjema()) {
+            const nyMottaker = felterTilSkjematBrevmottaker(useSkjema.skjema.felter);
+            const mottakere: SkjemaBrevmottaker[] = [...manuelleBrevmottakerePåFagsak, nyMottaker];
+            return settManuelleBrevmottakerePåFagsak(mottakere);
+        }
+    };
+
+    const fjernMottaker = (mottaker: SkjemaBrevmottaker) => {
+        const mottakereUtenFjernetPerson = manuelleBrevmottakerePåFagsak.filter(
+            it => it !== mottaker
```

Comment:
Litt kotlin-aktig?

Reply:
Fiksa i [Rename it -> mottakerPåFagsak](https://github.com/navikt/familie-ba-sak-frontend/pull/2860/commits/4fff8e45e11ee82cfd6ea739dd5f4a15c5dfb720)